utils/gdpr.js

```diff
@@ -0,0 +1,14 @@
+const GDPR_KEY = 'GDPR_accepted';
+
+export const getGDPRAccepted = () => {
+  if (typeof window !== 'undefined') {
+    return window.localStorage.getItem(GDPR_KEY) === 'true';
+  }
+  return true;
+};
+
+export const setGDPRAccepted = (value) => {
```

Comment:
If the only option is accepting the terms I found a bit useless passing a `value` attribute, just setting the key to `true` should be fine.